client/tailwind.config.cjs

```diff
@@ -5,7 +5,7 @@ module.exports = {
     extend: {}
   },
   daisyui: {
-    themes: ["cupcake", "dark"]
```

Comment:
I think we should keep a theme for dark mode (it doesn't have to be "dark" theme, but should have an alternative for people who prefer to view the app in dark mode)